tests/jobscript/12-err-script/suite.rc

```diff
@@ -8,9 +8,8 @@
         R1 = foo
 [runtime]
     [[foo]]
+        init-script = FOO="foo bar baz qux"
         err-script = echo "$1 ${FOO}"
```

Comment:
With this PR it is no longer possible to access user environment or script variables from `err-script` and `exit-script` (by design) and instead `init-script` has to be used for that.